src/SUMMARY.md

```diff
@@ -18,7 +18,7 @@
 - [Install with Docker](administration/install_docker.md)
 - [Install with Ansible](administration/install_ansible.md)
 - [Install from Scratch](administration/from_scratch.md)
-- [Install on Elestio](adminitstration/)
+- [Install on Elestio](administration/)
```

Comment:
You can get rid of this line as its part of the existing administration page now which is already linked.